alacritty/build.rs

```diff
@@ -22,6 +22,10 @@ use std::path::Path;
 use embed_resource;
 
 fn main() {
+    if cfg!(not(any(feature = "x11", feature = "wayland", target_os = "macos", windows))) {
+        panic!("At least one of the \"x11\"/\"wayland\" features must be enabled");
```

Comment:
```suggestion
        panic!("at least one of the \"x11\"/\"wayland\" features must be enabled");
```

Needs to be lowercase to follow Rust's formatting standards, see [here](https://doc.rust-lang.org/src/alloc/string.rs.html#1214) for an example.

Reply:
Or even, https://github.com/rust-lang/api-guidelines/issues/79#issuecomment-316275971

Reply:
Maybe we should use \`x11\` and \`wayland\` too? I'm not sure if that's standardized?

I was kinda holding out on that because I thought the panic message itself was surrounded with \`, but turns out I misremembered and it's actually '.

Reply:
I'd prefer to surround with `"` tbh, since ` is too markdownish.

Reply:
@kchibisov the quotes are from the `panic!` implementation, we don't have any control over that.

Reply:
@nixpulvis you don't get the issue. I'm talking about the quotes around `x11` and `wayland`, not about the thing panic inserts there, I'm fine with `'` from it.

Reply:
What's wrong with markdownish? Rust documentation uses markdown too.

Since we're talking about strings I think both would be fine (`Debug` for String uses `"` too I believe), but I'm not sure if that's the recommended strategy and uniformity is good here.

I have searched the Rust source code and there's no clear indication for me. There are some panics with `"`, a lot with `\`` but not all with strings obviously and some with `'` used for chars for example. So I think `"` is fine?

Reply:
Though we wouldn't need the \\", which might be a good argument already to switch to \`.

But I'll leave this up to you @kchibisov, I'd accept both. But I think I'd personally prefer \` I think.

Reply:
The problem I see with \` is that they are not that noticeable than `"` when you're trying to find things among all this output from panic, so I'm in favor of using more visible thing `"`. It's not only about generic panic, this panic is an artificial compilation error (read `#error` from C world).